apps/service-catalog/app/catalogs/[catalogId]/layout.tsx

```diff
@@ -0,0 +1,25 @@
+import { Layout, TermsOfUseModal } from '@catalog-frontend/ui';
+import { localization } from '@catalog-frontend/utils';
+
+const PageLayout = ({
+  children,
+  params: { catalogId },
+}: {
+  children: React.ReactNode;
+  params: { catalogId: string };
+}) => {
+  return (
+    <Layout
```

Comment:
Skulle denne vært CatalogLayout for å få med designet?

Reply:
Sannsynligvis, men tenker det da er bedre å fikse i en pr som tar seg tjenestekatalogene som en helhet. Som da også fikser feks detaljside og skjema

Reply:
Aha ja sorry, leste feil og trodde det var data-service 😅